module/VuFind/src/VuFind/View/Helper/Root/GoogleTagManager.php

```diff
@@ -0,0 +1,93 @@
+<?php
+/**
+ * GoogleTagManager view helper
+ *
+ * PHP version 7
+ *
+ * Copyright (C) Villanova University 2010.
```

Comment:
Might want to use 2022 here.

Reply:
Yes thanks, will correct.  How does IP work for VuFind -- do all contributors assign copyright to Villanova?

Reply:
The copyright does not have to be assigned to Villanova; as long as it's GPL'ed, we're not overly concerned whose name is on it. :-)